### ISSUE TITLE
Pin to distopia < 0.3 ahead of v2.8.0 release

### ISSUE THREAD

**Issue by IAlibay** [MDAnalysis/mdanalysis#4739]
Full deps CI is currentlly failling because of distopia's latest release.

We should pin the upper bound in the code checks to make sure that this doesn't hit users on release.

**Comment by orbeckst** [comment]
yes, eg PR #4737 is now failing due to this issue so we need at least a quick fix to be able to work on PRs.

**Comment by orbeckst** [comment]
Presumable PR #4734 should fix this?

**Comment by IAlibay** [comment]
> Presumable PR #4734 should fix this?

Yes but I don't think a WIP should be on the release milestone? At least not if we're planning a release at the end of the weekend.

**Comment by orbeckst** [comment]
That would seem ambitious and that was not what I wanted to suggest. I wanted to provide some context because it initially looked to me as if no-one was aware of what needs doing.

**Comment by orbeckst** [comment]
Oh, I think now I see how you understood my comment. Let me rephrase 

> Presumable PR https://github.com/MDAnalysis/mdanalysis/pull/4734 should fix this?

to

> Let's pin distopia<0.3.0 as suggested in this issue so that CI runs again and so that we can get 2.8.0 out on time. But I hope that PR https://github.com/MDAnalysis/mdanalysis/pull/4734 will soon be done so that we can then remove the pin and use the latest version of distopia (presumably it will have to be >= 0.3.0).
